rules/autoconsent/hema-com.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "hema",
+  "prehideSelectors": [".cookie-modal"],
+  "detectCmp": [
+    { "exists": ".cookie-modal" }
```

Comment:
This seems quite generic to me. Can we move the more specific rule from `detectPopup` to here too?

Reply:
I forgot adding the `runContext`. Added.

Reply:
@seia-soto sorry, there was a runContext in your initial version. I removed it so that this rule [works on other domains](https://github.com/duckduckgo/autoconsent/pull/319#issuecomment-1871157374) like hema.nl. I think the best way would be to remove the runContext and use the same selectors in `detectCmp` and `detectPopup`, but use "exists" instead of "visible" in `detectCmp`.

Reply:
No problem! :) Thanks for clearifying.

Reply:
I thought that this rule without runContext was too generic but using `visible` instead of `exists` makes sense to me.